### PR TITLE
fix alpha

### PR DESCRIPTION
**Pull request by jafu888**

### CODE REVIEW

File: constraintlayout/core/src/main/java/androidx/constraintlayout/core/motion/MotionWidget.java [hunk 1]
```diff
@@ -31,12 +31,12 @@ public class MotionWidget implements TypedValues {
...
     public static final int VISIBILITY_MODE_NORMAL = 0;
     public static final int VISIBILITY_MODE_IGNORE = 1;
-    @SuppressWarnings("unused") private static final int INTERNAL_MATCH_PARENT = -1;
-    @SuppressWarnings("unused") private static final int INTERNAL_WRAP_CONTENT = -2;
+    private static final int INTERNAL_MATCH_PARENT = -1;
+    private static final int INTERNAL_WRAP_CONTENT = -2;
     public static final int INVISIBLE = 0;
     public static final int VISIBLE = 4;
-    @SuppressWarnings("unused") private static final int INTERNAL_MATCH_CONSTRAINT = -3;
-    @SuppressWarnings("unused") private static final int INTERNAL_WRAP_CONTENT_CONSTRAINED = -4;
```
Comment (oscar-ad) [inline constraintlayout/core/src/main/java/androidx/constraintlayout/core/motion/MotionWidget.java:39]: It doesn't seem like these are being used.

File: constraintlayout/core/src/main/java/androidx/constraintlayout/core/motion/MotionWidget.java [hunk 2]
```diff
@@ -70,8 +70,8 @@ public static class Motion {
         public String mQuantizeInterpolatorString = null;
         public int mQuantizeInterpolatorType = INTERPOLATOR_UNDEFINED; // undefined
         public int mQuantizeInterpolatorID = -1;
-        @SuppressWarnings("unused") private static final int INTERPOLATOR_REFERENCE_ID = -2;
-        @SuppressWarnings("unused") private static final int SPLINE_STRING = -1;
```
Comment (oscar-ad) [inline constraintlayout/core/src/main/java/androidx/constraintlayout/core/motion/MotionWidget.java:74]: Likewise, they remain unused.

**Review by oscar-ad — CHANGES_REQUESTED** [review]

File: constraintlayout/core/src/main/java/androidx/constraintlayout/core/motion/MotionWidget.java (hunk 1 above)
Comment (jafu888) [reply constraintlayout/core/src/main/java/androidx/constraintlayout/core/motion/MotionWidget.java:39]: This was a rebase accident removing the changes

File: constraintlayout/core/src/main/java/androidx/constraintlayout/core/motion/MotionWidget.java (hunk 2 above)
Comment (jafu888) [reply constraintlayout/core/src/main/java/androidx/constraintlayout/core/motion/MotionWidget.java:74]: This was a rebase accident removing the changes

**Review by oscar-ad — APPROVED** [review]

**Pull request merged**